src/shared/sam/cli/samCliLocalInvoke.ts

```diff
@@ -30,6 +32,8 @@ export interface SamLocalInvokeCommandArgs {
     /** Wait until strings specified in `debuggerAttachCues` appear in the process output.  */
     waitForCues: boolean
     timeout?: Timeout
+    /** Allows us to name debug sessions so we can terminate them later */
+    name?: string
```

Comment:
good idea! Is this easier that sharing the id I guess?

Reply:
ID could be used too but name felt more natural (and yes easier too)